NCEP/gdas_utility.py

```diff
@@ -39,19 +39,24 @@ def __init__(self, start_datetime, end_datetime, num_pressure_levels=13, downloa
         if self.download_source == 's3':
             # Initialize the S3 client
             # Specify the path to your custom AWS credentials file
-            custom_credentials_file = '/contrib/Sadegh.Tabas/.aws/credentials'
+            #custom_credentials_file = '/contrib/Sadegh.Tabas/.aws/credentials'
```

Comment:
@LinlinCui-NOAA could you please return these, if we want to import credentials then we need to update job cards as well which is not needed. Please just return the commands for this as it was before. Thanks

Reply:
Also, I was wondering if there was any issue with the "NOMADS" option. Thanks